Robust/Service/IWindowService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using Robust.Model.Product;
+using Robust.View;
+using Robust.ViewModel;
+
+namespace Robust.Service
```

Comment:
Siden denne her nok kommer til at have alle vores interfaces kan vi lave namespacet mere specifik såsom

Robust.Service.Interface

Kig igennem for en forklaring på hvorfor vi skal have mere konkrete namespaces